universal-application-tool-0.0.1/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -93,11 +93,28 @@ public boolean hasErrors() {
   }
 
   public Optional<Long> getUpdatedInProgramMetadata() {
-    return applicantData.readLong(getContextualizedPath().join(Scalar.PROGRAM_UPDATED_IN));
+    Path contextualizedMetadataPath = getContextualizedPath().join(Scalar.PROGRAM_UPDATED_IN);
+
+    // Metadata for enumerators is stored for each JSON array element, but we rely on metadata for
+    // the first one.
```

Comment:
question: to clarify, this is *just* for the enumerator question, correct? A repeated question will have its own metadata separate from the entity?

Reply:
yes

Reply:
Thanks!